storage-service/src/main/resources/db/migration/V1.0.0__init.sql

```diff
@@ -4,5 +4,6 @@ CREATE TABLE repository
 (
   id            BIGINT PRIMARY KEY,
   repourl       VARCHAR(100) NOT NULL UNIQUE,
-  name          VARCHAR(20)
+  name          VARCHAR(100),
```

Comment:
What if there were persistent instances of the storage? Would this work? My guess is that in such case this script would be already executed in the DB and when trying to run the migration, Flyway would throw an error caused by mismatch of the digest of this script.

Reply:
You are right. I modified this script because at the moment, we didn't have real data. From now on, we can't update these scripts, but add incremental ones: that's why I also added the versions 1.0.1 and 1.0.2.